.gitignore

```diff
@@ -0,0 +1 @@
+*.swp
```

Comment:
You might want to consider using a [global .gitignore](https://help.github.com/articles/ignoring-files/#create-a-global-gitignore) for $EDITOR specific files.

Reply:
This is a default file helm created for me... :) but fair point I think we would need to discuss it for all projects.